docs/state-sync.md

```diff
@@ -0,0 +1,129 @@
+# State Sync
+
+The Provenance 1.10.0+ release comes with Tendermint Core 0.34 which includes support for state sync. State sync allows a new node to join a network by fetching a snapshot of the application state at a recent height vs. fetching and replaying all historical blocks (which can take days). This can reduce the time needed to sync with the network down to minutes.
+
+<!-- TOC -->
+- [Prerequisites](#prerequisites)
+    - [provenanced binary](#provenanced-binary)
+    - [data directory](#data-directory)
+    - [Starting a new node from State Sync](#starting-a-new-node-from-state-sync)
+        - [Mainnet](#mainnet)
+        - [Testnet](#testnet)
+        - [Power users](#power-users)
```

Comment:
These are all out of whack. Just use `make update-tocs` for this (or `scripts/update-toc.sh docs`).

This is what it should look like:
```plaintext
<!-- TOC -->
  - [Prerequisites](#prerequisites)
    - [provenanced binary](#provenanced-binary)
    - [data directory](#data-directory)
  - [Starting a new node from State Sync](#starting-a-new-node-from-state-sync)
    - [Mainnet](#mainnet)
    - [Testnet](#testnet)
    - [Power users](#power-users)
```

Reply:
done.